routers/web/user/setting/profile.go

```diff
@@ -357,6 +358,22 @@ func Appearance(ctx *context.Context) {
 	ctx.Data["Title"] = ctx.Tr("settings")
 	ctx.Data["PageIsSettingsAppearance"] = true
 
+	hiddenEvents := &forms.UpdateCommentTypeForm{}
+	eventsRaw, err := user_model.GetSettings(ctx.User.ID, []string{"hidden_comment_types"})
```

Comment:
`hidden_comment_types` appears too many times, we need some constants, or we should use some helper functions to maintain the HiddenCommentTypes settings for users.

Reply:
I don't really know, but if we would add a new helper function with every new setting, this would add a bit unnecessary code. I would be for just using the constants.

Reply:
I agree with @wxiaoguang here. We can't just repeat this `hidden_comment_types` all over the code. 

At the very least we need a constant.

You need to find somewhere that you can put a constant, probably routers/web/issue/

Reply:
https://github.com/go-gitea/gitea/pull/17414/commits/ff70bdb7505fbdee66cb88bb6f468ac9540bdfce adds a variable to fix this